webapp/src/ts/services/target-aggregates.service.ts

```diff
@@ -315,19 +322,20 @@ export class TargetAggregatesService {
     return !facilityIds || facilityIds.length > 0;
   }
 
-  getReportingMonth(reportingPeriod) {
+  getReportingMonth(reportingPeriod:ReportingPeriod) {
     return this.settingsService
       .get()
       .then(settings => {
-        const tag = this.getIntervalTag(settings, reportingPeriod);
-        return moment(tag, 'YYYY-MM').format('MMMM');
+        const tag = this.getTargetIntervalTag(settings, reportingPeriod);
+        return moment(tag, this.INTERVAL_TAG_FORMAT).format('MMMM');
```

Comment:
Why it's changing from 'YYYY-MM' to 'Y-MM'?

Reply:
I wanted less magic strings. I'm unsure why there was a difference in the first place.

Reply:
Is at the end the same thing `YYYY-MM` to `Y-MM` for the usage in targets? 
If yes, I think `YYYY-MM` is something more people will understand when looking at it.

Reply:
it's the same thing.